grpc_check/manifest.json

```diff
@@ -10,7 +10,12 @@
     "changelog": "CHANGELOG.md",
```

Comment:
Hi @fshaik3494 , this looks like a file from a different integration. Can you revert all changes to this file?